tests/test_data/TransformedComponent.step

```diff
@@ -0,0 +1,588 @@
+ISO-10303-21;

```

Comment:
Would this file have triggered the assert previously?   I guess it gets loaded and used for lots of tests, but maybe using the function which just splits the bodies.

Reply:
> Do you use any ugrid features?

Since we are using an old version of UV-Net we do not use the ugrid features at all.

> Would this file have triggered the assert previously?

It caused two test failures both in `GridTester`